src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html

```diff
@@ -663,7 +665,7 @@ <h4 *ngIf="displaySubsection(0, 10)">Setting Up Questions</h4>
       </li>
     </ul>
     <p>
-      Technical details about how ranks are calculated are available <a (click)="openRankInfoModal(rankInfoHelp)" [tmRouterLink]="">here</a>.
+      Technical details about how ranks are calculated are available <span class=modal-window-class> <a (click)="openRankInfoModal(rankInfoHelp)" [tmRouterLink]="">here</a></span>.
```

Comment:
No exactly sure how this part works without `ngTooltip` yet still showing the tool tip.

Reply:
That was my bad, I had the local copy that made that and I was putting everything together in a segmented fashion and I thought I had pushed up the changes, I did not realize I made that mistake, I am so sorry. The new commit has the changes for the second modal link

Reply:
No worries. This is what code review is for. It is fine to have some git mistake.